### ISSUE TITLE
Load MNIST dataset with opencv

### ISSUE THREAD

**Issue by Jeli04** [nsnod/CTorch#22]
Load all MNIST images into a tensor of (1x784) where 32x32 = 784. Split into a train and test set as well. Verify the data in the tensor is correct by comparing with PyTorch version (compare the raw numbers of two of the same images). Be sure to apply the proper standard normalization as well.
